pom.xml

```diff
@@ -551,6 +614,21 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.platform</groupId>
+      <artifactId>junit-platform-commons</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
For some of these (I'm only explicitly calling out these JUnit ones because I know we've switched to using the BOM), I don't know if we explicitly want to add them into the pom or use one the additional configs that can be added to the `maven-dependency-plugin`.
https://maven.apache.org/plugins/maven-dependency-plugin/examples/exclude-dependencies-from-dependency-analysis.html
Using something like `<ignoredUsedUndeclaredDependencies>` and setting these in there may be good since they are all included under the umbrella of the BOM, and _technically_ then aren't declared but used.

I'm not 100% if these other config tabs would be accurate to use for others of theses dependencies or not, but wanted to at lease throw this idea out.

Reply:
I like where you are going with that, and I agree we might want to clean this up a bit. I figured we would do this step first and then we can follow-up with configuring the tool better?